README.md

```diff
@@ -1,3 +1,14 @@
+## Links for Task2  
+
+Manual configured S3/CloudFrond resources:  
+S3 link - http://rss-aws-app.s3-website-eu-west-1.amazonaws.com/  
+CloudFront link - https://d3rdi212t1dvge.cloudfront.net/  
+
+Automate configured S3/CloudFrond resources:  
+S3 link - http://rss-second-aws-app.s3-website-eu-west-1.amazonaws.com/  
+CloudFront link - https://d3gqyf3iecrlqi.cloudfront.net  
```

Comment:
This link doesn't work. I'm getting an error:
```xml
<Error>
<Code>AuthorizationHeaderMalformed</Code>
<Message>The authorization header is malformed; the region 'us-east-1' is wrong; expecting 'eu-west-1'</Message>
<Region>eu-west-1</Region>
<RequestId>33067E35BFF5260B</RequestId>
<HostId>gOakwBhPt13IEEr7Ghrgs/XTmzaU9LO+VKP5G6AQ44kAXLkuevGKRG//0EqT0uIWdT2h1iA7Caw=</HostId>
</Error>
```

Reply:
You probably moved your bucket to eu-west-1 after having initially deployed it to us-east-1. Try re-deploying the CF stack. You may want to update this link afterwards.

Reply:
Eventually, a couple a days later, link was recovered by Amason. Now it's working.